company-profiles/brikl.md

```diff
@@ -0,0 +1,33 @@
+# Brikl
+
+## Company blurb
+
+Brikl is the leading eCommerce MicroStore technology partner for custom and promotional businesses worldwide. Our market-leading features include 15-minute store setup, bulk embellishment, and supplier integrations, vastly reducing the effort, cost, and chaos involved in the selling and customization process. We help build profitable businesses, increase employee happiness and offer internal efficiencies that positively impact the planet, enabling companies to create better across their organizations.
+
+## Company size
+
+100-150 Employees
+
+## Remote status
+
+5 days work week with flexible hours and WFH policy.
+
+## Region
+
+North America, Asia
+
+## Company technologies
+
+Coalition is the leading metrics-driven digital agency. We specialize in SEO, PPC, email marketing, social media marketing, and web design & development.
```

Comment:
I think this may be incorrect as the company name is `Brikl` rather than `Coaltion`

Reply:
Oh my bad it's copy monster will change it right away!